app/assets/stylesheets/preact/article-form.scss

```diff
@@ -555,6 +555,7 @@
       max-width: 24px;
       margin: -10px 5px;
       position: relative;
+      filter: invert(100%);
```

Comment:
no need for the `themeable` stuff here as the background of the image overlay in the editor is always dark regardless of the theme. Let me know if I should add it nonetheless